src/components/shared/Entity/Actions/TestButton.tsx

```diff
@@ -0,0 +1,62 @@
+import { EditorStoreState } from 'components/editor/Store';
+import EntityCreateSave from 'components/shared/Entity/Actions/Save';
+import LogDialog from 'components/shared/Entity/LogDialog';
+import LogDialogActions from 'components/shared/Entity/LogDialogActions';
+import { DraftSpecQuery } from 'hooks/useDraftSpecs';
+import { useRouteStore } from 'hooks/useRouteStore';
+import { useZustandStore } from 'hooks/useZustand';
+import { FormattedMessage } from 'react-intl';
+import { entityCreateStoreSelectors, FormStatus } from 'stores/Create';
+
+interface Props {
+    closeLogs: Function;
+    callFailed: Function;
+    disabled: boolean;
+}
+
+function EntityTestButton({ callFailed, closeLogs, disabled }: Props) {
```

Comment:
Test button is just a wrapper for the save button (set to dry run) and the test progress dialog